package/libcurl/package

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# Copyright (c) 2021 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(libcurl-dev)
+pkgdesc="Client-side URL transfer library"
+url=https://curl.se/libcurl
+pkgver=7.75.0-1
+timestamp=2021-04-14T00:00:00Z
+section="devel"
+maintainer="TBD"
```

Comment:
Did you forget to add the maintainer this or do you prefer for someone else to be the formal maintainer of this package?

Reply:
According to the docs we should be using `None <none@example.org>` instead of `TBD`
https://github.com/toltec-dev/toltec/blob/testing/docs/package.md#maintainer-field

Reply:
I'd prefer for someone else to be the formal maintainer, so I left it invalid on purpose.
I can change it to None, but I assumed it wouldn't be merged without a proper maintainer anyway?

Reply:
No problem. I didn't know until yet either, but using something like `None <none@example.org>` (from [docs here](https://github.com/toltec-dev/toltec/blob/stable/docs/package.md#maintainer)) seems to be exactly what you need.

I had also added some packages not developed myself (namely [evtest](https://github.com/toltec-dev/toltec/blob/stable/package/evtest/package)) and just added myself. I think of it more like being the spokesperson if any specific issues. So people know who to tag in that case. But I find it perfectly fine to not want to do that as well. :+1: 

So just use something like "None" since it is specified in the docs that way.